docs/en/observability/synthetics-migrate-integration.asciidoc

```diff
@@ -0,0 +1,167 @@
+
+* Your infrastructure using <<synthetics-private-location,private locations>>
+* Elastic's global managed infrastructure
+
+| *Where you configure monitors*
+a| You could configure monitors using:
+
+* A user interface in {kib} (all monitor types)
+* Code in an external, version-controlled project (browser monitors _only_)
+a| You can configure monitors using:
+
+* A user interface in {kib} (all lightweight monitors, browser monitors via inline script only)
+* Code in an external, version-controlled project (all monitor types)
+
+| *How to use projects*
+(<<synthetics-migrate-integration-projects,read{nbsp}more>>)
+a| . Created a project that uses `@elastic/synthetics`.
+   . Wrote journeys in JavaScript or TypeScript files.
+   . Zipped the entire project.
+   . Configured and created the monitor in the
+     Integrations UI by adding a ZIP URL that pointed to
+     the location of the project.
+a| . Create a project that uses `@elastic/synthetics`.
+   . Configure lightweight monitors in YAML files.
+   . Write journeys in JavaScript or TypeScript files and configure
+     individual monitors in your journey code using `monitor.use` or
+     configure all monitors using the `synthetics.config.ts` file.
+   . Use the `elastic/synthetics push` command to create monitors.
+
+Find more details in <<synthetics-get-started-project>>.
+
+| *How to use the UI*
+(<<synthetics-migrate-integration-ui,read{nbsp}more>>)
+a| . Went to the *Integrations* page in {kib}.
+   . Searched for and added the *Elastic Synthetics* integration.
+   . Configured the monitor.
+   . Created the monitor.
+a| . Go to the *{uptime-app}* in {kib}.
+   . Go to *Monitor Management*.
+   . Click *Add monitor*.
+   . Configure the monitor.
+   . Create the monitor.
+
+Find more details in <<synthetics-get-started-ui>>.
+|===
+
+[discrete]
+[[synthetics-migrate-integration-location]]
+= Where monitors run
+
+If you want to continue hosting on your infrastructure, you will need to create a
+private location before creating monitors.
+
+// Is this true?
+If you have already have an {agent} running using `elastic-agent-complete`,
+you can <<synthetics-private-location-add,add it as a new private location>>
+in the {uptime-app}.
+
+To create a new private location from scratch, follow all instructions in
+<<synthetics-private-location>>.
+
+Alternatively, you can start hosting on Elastic's global managed infrastructure.
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations without having to manage your own infrastructure.
+Elastic takes care of software updates and capacity planning for you.
+
+[discrete]
+[[synthetics-migrate-integration-projects]]
+= How to use projects
+
+If you already have an external project you were adding via a ZIP URL
+you can use the same project, but you will have to make some changes.
+
+First, upgrade the existing project to use the latest version of 
+`@elastic/synthetics`:
```

Comment:
**Option 1: Warn users about the impact of running `init` on an existing project**

```suggestion
If you already have an external project you were adding via a ZIP URL
you can use the same project, but you will have to make some changes.

First, upgrade the existing project to use the latest version of 
`@elastic/synthetics`.

[IMPORTANT]
====
*Upgrading your project using these instructions will overwrite your existing synthetics configuration file (`synthetics.config.ts` or `synthetics.config.js`) and `package.json` file.*

If you have customized either of these files in your existing project, be sure you
have the current version saved in version control or create a fresh project using
`npm @elastic/synthetics init` with a path to a new directory instead of the path to
existing project. Then, manually transfer:
* Any synthetics configuration options to the new synthetics configuration file.
* Any dependencies, scripts, or other configurations to the new `package.json` file.
====
```